src/main/resources/static/domain-events-api.yml

```diff
@@ -85,6 +85,33 @@ paths:
                 $ref: '#/components/schemas/Problem'
         500:
           $ref: '#/components/responses/500Response'
+  /events/application-appeal-made/{eventId}:
+    parameters:
+      - name: eventId
+        description: UUID of the event
+        in: path
+        required: true
+        schema:
+          $ref: '#/components/schemas/EventId'
+    get:
+      tags:
+        - "'Apply' events"
+      summary: An 'application-appeal-made' event
```

Comment:
I think something like `assessment.appealed` is  more what we need here in order to reflect the fact that it's the *assessment* which is being appealed, rather than the application (see https://dsdmoj.atlassian.net/browse/APS-239?focusedCommentId=420234)

Reply:
This PR has now been refactored to reflect the new name.